package.json

```diff
@@ -10,16 +10,21 @@
     "url": "shelf.io"
   },
   "engines": {
-    "node": ">=8"
+    "node": ">=12"
   },
   "scripts": {
     "coverage": "jest --coverage",
     "lint": "eslint . --ext .js,.ts,.json --fix",
-    "lint:ci": "eslint . --ext .js,.ts,.json",
-    "test": "jest"
+    "lint:ci": "eslint . --ext .js,.ts,.json --quiet",
+    "test": "jest src",
+    "prepack": "yarn build",
+    "type-check": "tsc --noEmit",
+    "type-check:watch": "npm run type-check -- --watch",
+    "build": "rm -rf lib/ && yarn build:types && babel src --out-dir lib --ignore '**/*.test.ts' --extensions '.ts'",
+    "build:types": "tsc --emitDeclarationOnly --declaration --isolatedModules false --declarationDir lib"
   },
   "files": [
-    "index.js"
+    "lib"
   ],
```

Comment:
See https://github.com/shelfio/fast-chunk-string/commit/ba8dab183b6c0b4798a46d54890dbb5fbf015d27, it was forgotten last time